dallinger/prolific.py

```diff
@@ -674,8 +674,13 @@ def prolific_service_from_config():  #
 
 
 def dev_prolific_service_from_config():
+    from dallinger.config import get_config
     from dallinger.prolific import DevProlificService
 
+    config = get_config()
+    config.load()
+    config.set("prolific_workspace", "62fce6fff0a78eb4f3ebc09c")
```

Comment:
Unfortunately I don't think we should use `config.set` as it can introduce subtle bugs, since the updated config is not shared across processes. I think config should be considered immutable. Can you think of another way around this?